HPXMLtoOpenStudio/resources/defaults.rb

```diff
@@ -6111,102 +6124,133 @@ def self.get_water_heater_heating_capacity(fuel, nbeds, num_water_heaters, nbath
     # Adjust the heating capacity if there are multiple water heaters in the home
     nbaths /= num_water_heaters.to_f
 
-    if fuel != HPXML::FuelTypeElectricity
-      if nbeds <= 3
-        cap_kbtuh = 36.0
-      elsif nbeds == 4
-        cap_kbtuh = 38.0
-      elsif nbeds == 5
-        cap_kbtuh = 48.0
-      else
-        cap_kbtuh = 50.0
-      end
-      return cap_kbtuh
-    else
-      if nbeds == 1
```

Comment:
Hard to see due to the code refactor, but here's the bugfix. This should have been `nbeds <= 1`, not `nbeds == 1` to accommodate studios.